### DISCUSSION TITLE
Tailwind CSS

### DISCUSSION THREAD

**Discussion by dbnar2** [refinedev/refine#1448] [Q&A] [answered]
Is this framework able to integrate nicely with Tailwind CSS?

**Comment by omeraplak** [comment]
Hey,
Yes, you can integrate it. A headless version of Refine will be released very soon. Our official headless demo also uses Tailwind!

https://github.com/pankod/refine/tree/alpha/examples/fineFoods/client

Live Demo: https://alpha-client.refine.pankod.com

**Comment by omeraplak** [comment, marked as answer]
Hey @dbnar2 ,
We've released official Tailwind CSS tutorial -> https://refine.dev/docs/core/tutorial/

**Reply by 5amfung** [reply]
@omeraplak The link leads to 404.

**Reply by omeraplak** [reply]
Hey @5amfung ,
Thank you for contacting us. You can install TailwindCSS in refine applications the same way you would in a normal React application. Please refer to https://tailwindcss.com/docs/installation/framework-guides for instructions.

**Comment by Nicholas0350** [comment]
Hey @omeraplak 

All links relative to this question are 404's

**Reply by omeraplak** [reply]
Hey @Nicholas0350 ,
Thank you for contacting us. You can install TailwindCSS in refine applications the same way you would in a normal React application. Please refer to https://tailwindcss.com/docs/installation/framework-guides for instructions.
